### PR TITLE
fix: 🐛 mew address select checksum

### PR DESCRIPTION
**Pull request by gesseekur**
### Bug

* \[x] Updated CHANGELOG.md
* \[x] Add PR label

https://app.asana.com/0/1163629870679883/1200606723374661

### CODE REVIEW

**Review by gamalielhere — APPROVED** [review]

**Review by rsteubs — APPROVED** [review]

File: src/modules/address-book/components/AddressBookAddEdit.vue
Comment (kvhnuke) [inline src/modules/address-book/components/AddressBookAddEdit.vue:156]: why are we checking for `isHexStrict`?

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: src/modules/address-book/components/AddressBookAddEdit.vue
Comment (gesseekur) [reply src/modules/address-book/components/AddressBookAddEdit.vue:156]: because of this bug https://app.asana.com/0/1163629870679883/1200627642947378 hahah should we just allow them to add those addresses?

Comment (kvhnuke) [reply src/modules/address-book/components/AddressBookAddEdit.vue:156]: yea but isAddress should cover it

**Review by kvhnuke — CHANGES_REQUESTED** [review]

File: src/modules/address-book/components/AddressBookAddEdit.vue
Comment (gesseekur) [reply src/modules/address-book/components/AddressBookAddEdit.vue:156]: hmmm when I tried isAddress on there , it returned true but i can try again

Comment (gesseekur) [reply src/modules/address-book/components/AddressBookAddEdit.vue:156]: its returning true for 1254155464511115555555555555555555555511 D:

Comment (kvhnuke) [reply src/modules/address-book/components/AddressBookAddEdit.vue:156]: weird, k let me take a look

Comment (kvhnuke) [reply src/modules/address-book/components/AddressBookAddEdit.vue]: > its returning true for 1254155464511115555555555555555555555511 D:

yea you are right, https://web3js.readthedocs.io/en/v1.4.0/web3-utils.html#id32

I changed it to our custom isAddress, which check for isHexStrict. Can you add a debounce to name resolving right now it tries to resolve on every keypress after `.`

**Review by hsurf22 — APPROVED** [review]

File: src/modules/address-book/components/AddressBookAddEdit.vue
Comment (gesseekur) [reply src/modules/address-book/components/AddressBookAddEdit.vue]: yep! after playing around with it , i think i need to change something from components first

Comment (gesseekur) [reply src/modules/address-book/components/AddressBookAddEdit.vue]: helloo I pushed something new and updated components but I can't update the package here yet until the overlay changes are approved , will update after!

**Comment by gamalielhere** [comment]
Moved changes to #3377

**Pull request closed without merging**
